mongodbatlas/fw_common.go

```diff
@@ -0,0 +1,68 @@
+package mongodbatlas
```

Comment:
Does it make sense to move this struct in a subdir to make it clear that does not define a resource or ds?

Reply:
Had looked into this but reached a limitation. When defining `fw_common.go` within framework package, I need to import mongodbatlas package to reference `MongoDBClient`. This leads to an `import cycle not allowed` error.